macros/compare_all_columns.sql

```diff
@@ -0,0 +1,51 @@
+{% macro compare_all_columns( a_relation, b_relation, exclude_columns, primary_key ) -%}
+  {{ return(adapter.dispatch('compare_all_columns', 'audit_helper')( a_relation, b_relation, exclude_columns, primary_key )) }}
+{%- endmacro %}
+
+{% macro default__compare_all_columns( a_relation, b_relation, exclude_columns, primary_key ) -%}
```

Comment:
I think it'd be worthwhile having a `summarize` option here too, in the same spirit as #52